app/models/current.rb

```diff
@@ -0,0 +1,3 @@
+class Current < ActiveSupport::CurrentAttributes
+  attribute :user
+end
```

Comment:
what is the purpose of this model?

Reply:
Bhaia, by subclassing `ActiveSupport::CurrentAttributes` and defining attributes within the subclass, we can store data that is specific to the current request, such as the `current user`, and access it from anywhere within our Rails application during that request's processing.

Reply:
query: Why you choose this instead of a simple helper function?